shell/platform/darwin/macos/framework/Source/FlutterCompositor.h

```diff
@@ -76,15 +78,18 @@ class FlutterCompositor {
   bool EndFrame(bool has_flutter_content);
 
   // Creates a CALayer object which is backed by the supplied IOSurface, and
-  // adds it to the root CALayer for this FlutterViewController's view.
+  // adds it to the root CALayer for the given view.
   void InsertCALayerForIOSurface(
+      FlutterView* view,
       const IOSurfaceRef& io_surface,
       CATransform3D transform = CATransform3DIdentity);
 
  private:
   // A list of the active CALayer objects for the frame that need to be removed.
   std::list<CALayer*> active_ca_layers_;
 
+  ViewProvider get_view_callback_;
```

Comment:
Ideally this should have a declaration comment explain what its role is as well.